node/node_v2_test.go

```diff
@@ -60,36 +52,11 @@ func TestDownloadBundles(t *testing.T) {
 	require.Equal(t, blobCerts[0], blobShards[0].BlobCertificate)
 	require.Equal(t, blobCerts[1], blobShards[1].BlobCertificate)
 	require.Equal(t, blobCerts[2], blobShards[2].BlobCertificate)
-	require.Contains(t, blobShards[0].Bundles, core.QuorumID(0))
-	require.Contains(t, blobShards[0].Bundles, core.QuorumID(1))
-	require.Contains(t, blobShards[1].Bundles, core.QuorumID(0))
-	require.Contains(t, blobShards[1].Bundles, core.QuorumID(1))
-	require.Contains(t, blobShards[2].Bundles, core.QuorumID(1))
-	require.Contains(t, blobShards[2].Bundles, core.QuorumID(2))
-	bundleEqual(t, bundles[0][0], blobShards[0].Bundles[0])
-	bundleEqual(t, bundles[0][1], blobShards[0].Bundles[1])
-	bundleEqual(t, bundles[1][0], blobShards[1].Bundles[0])
-	bundleEqual(t, bundles[1][1], blobShards[1].Bundles[1])
-	bundleEqual(t, bundles[2][1], blobShards[2].Bundles[1])
-	bundleEqual(t, bundles[2][2], blobShards[2].Bundles[2])
 
```

Comment:
You've removed all usages of `bundleEqual` and `frameEqual`, which could now be deleted